hack/test-sh.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+#
+# Copyright 2019 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+cd "${REPO_ROOT}"
+find . -type f -name '*_test.sh' -exec {} +
```

Comment:
I don't think that's doing what you think it does.
Assuming there are the files `00_test.sh`, `01_test.sh` and `02_test.sh`, this would call something like
```
00_test.sh 01_test.sh 02_test.sh
```
... it would call one script with all the other file names as arguments ... while we'd want to something like
```
00_test.sh
01_test.sh
02_test.sh
```
... each script being called individually with no arguments

Reply:
This should be good now. I switched back to xargs but a pretty simple implementation of it. Parse on the \00. I ran both make test-sh and hack/test-sh and they showed a  good exist status on their own, and failed with a `_test.sh` file that exits 1